sdk/keyvault/Azure.Security.KeyVault.Administration/src/KeyVaultAccessControlClient.cs

```diff
@@ -36,7 +36,7 @@ protected KeyVaultAccessControlClient()
         /// <summary>
         /// Initializes a new instance of the <see cref="KeyVaultAccessControlClient"/> class for the specified vault.
         /// </summary>
-        /// <param name="vaultUri">A <see cref="Uri"/> to the vault on which the client operates. Appears as "DNS Name" in the Azure portal.</param>
+        /// <param name="vaultUri">A <see cref="Uri"/> to the vault on which the client operates. Appears as "DNS Name" in the Azure portal. You should validate that this URI references a valid Key Vault or Managed HSM resource. See https://aka.ms/azsdk/blog/vault-uri for details.</param>
```

Comment:
>https://aka.ms/azsdk/blog/vault-uri [](http://example.com/codeflow?start=233&length=35)

This link does not exist.

Reply:
It will, which is why I allocated the link early. It won't be live till next Tuesday.